### ISSUE TITLE
Compile tor-static for Windows on Linux

### ISSUE THREAD

**Issue by lu4p** [cretz/tor-static#22]
Can tor-static be compiled on Linux for Windows? (without wine or vm)

**Comment by cretz** [comment]
Not really, or at least I don't know how. It uses gcc and other Linux features, but it leverages MinGW libs. You might be able to finagle it. But I am planning on releasing static Windows libs w/ the next release now that I have an AppVeyor CI up and running.

**Comment by lu4p** [comment]
@cretz still planning to distribute precompiled libs?

**Comment by cretz** [comment]
@lu4p - I would love to ideally, but I haven't been able to work on this project in a while due to other priorities. Ideally I would get tor-static up and going w/ 0.4.x and the travis/appveyor builds would automatically put the binaries in the release when tagged. Sorry; hopefully at some point I can revisit this and Bine and shore things up. I saw your PR and will comment there, thanks!
